sharedbuffers/mapped_struct.py

```diff
@@ -164,7 +167,8 @@ def _likebuffer(buf):
     """
     Takes a buffer object as parameter and returns a writable object with buffer protocol.
     """
-    if type(buf) is buffer or type(buf) is bytearray or type(buf) is bytes or isinstance(buf, bytes):
+    if type(buf) is buffer or type(buf) is bytearray or type(buf) is bytes or (
```

Comment:
Why the parenthis on the last 2 possible values? The opening ( should start when the if starts.